Based on:

> Allow a slightly wider range of valid vt numbers. Forward-ported from Gentoo
> Busybox 1.1.3.

> The previous spin of this patch on 1.1.3 had a 'wait(NULL);' right before
> return EXIT_SUCCESS. I don't think it's needed anymore, so I left it out.

--- a/console-tools/openvt.c	2010-11-22 22:24:58.000000000 +0200
+++ b/console-tools/openvt.c	2010-11-29 15:32:18.000000000 +0200
@@ -124,7 +124,7 @@ int openvt_main(int argc UNUSED_PARAM, c
 
 	if (flags & OPT_c) {
 		/* Check for illegal vt number: < 1 or > 63 */
-		vtno = xatou_range(str_c, 1, 63);
+		vtno = xatou_range(str_c, 0, 63);
 	} else {
 		vtno = find_free_vtno();
 	}
